controllers/disruption_controller.go

```diff
@@ -652,12 +648,20 @@ func (r *DisruptionReconciler) selectTargets(instance *chaosv1beta1.Disruption)
 
 // getChaosPods returns chaos pods owned by the given instance and having the given labels
 func (r *DisruptionReconciler) getChaosPods(instance *chaosv1beta1.Disruption, ls labels.Set) ([]corev1.Pod, error) {
-	ownedPods := make([]corev1.Pod, 0)
 	pods := &corev1.PodList{}
 
+	if ls == nil {
+		ls = make(map[string]string)
+	}
+
+	// If these aren't valid labels, we will instead list ALL pods
```

Comment:
This is creepy, do you have any concrete example of such a thing happening?

Reply:
Yes (this is the same bug we hit in December). If you do `disruption-namespaced-name=namespace/name`, the `/` character isn't valid, and it will list all pods. confirmed locally. But we can guarantee that `namespace` and `name` only have legal characters, so this is just a warning for the next person who changes this code